src/excmds.ts

```diff
@@ -2171,6 +2171,100 @@ export function unset(...keys: string[]) {
     config.unset(...target)
 }
 
+/** Toggle css style to a page.
+ *
+ * Syntax: style {mode} [...args]
+ *
+ * @param mode
+ *         ALIASES          MODES
+ *      styleinclude   : style -i
+ *      styledelete    : style -x
+ *      styledisable   : style -d
+ *      styleenable    : style -e
+ *      styletoggle    : style -t
+ *
+ * @param args `-name {name} -match {[prefix]|[domain]|[URL]} -css {[complete css block of code]}`
+ *
+ */
+//#background
+export async function style(mode: string, ...args: string[]) {
+    let name: string
+    let css: string
+    let append: boolean = false
+    //let cssregex = /(\S*\s+)\{((?:.|\n)*?)\}\s*$/gm
+    if (args.length > 0) {
+        if (args.includes("-name")) {
+            name = args[args.indexOf("-name") + 1]
+        } else throw 'Missing "-name" argument.'
```

Comment:
Minor point here - I think we probably want `--name` style arguments rather than `-name`. (I know `winopen -private` doesn't follow that rule... that's my fault :) ).

Reply:
That's totally ok with me since it follows the way Linux environment is and which I'm used to. If you want I could change -private too but that probably has big impact on users and would have to be in another PR.

Reply:
> If you want I could change -private too but that probably has big impact on users and would have to be in another PR.

No - you're right, please don't do that yet. I'll probably hack in a special case later once we have the arg parser.